network-store-server/src/main/resources/iidm.cql

```diff
@@ -292,7 +292,7 @@ CREATE MATERIALIZED VIEW IF NOT EXISTS iidm.switchByVoltageLevel AS
         PRIMARY KEY (networkUuid, voltageLevelId, id);
 
 CREATE TYPE iidm.phaseTapChangerStep (
-    position int,
+    position int, // TO REMOVE
```

Comment:
why we keep it ?

Reply:
Just to avoid scheme migration at next deployment. I will clean everything later.